README.md

```diff
@@ -148,31 +136,32 @@ Determines how to handle installing updates to the rbenv. There are currently 2
 
 The default is `"none"`.
 
-### root_path
+## root_path
 
 The path prefix to rbenv in a system-wide installation.
 
 The default is `"/usr/local/rbenv"`.
 
-### patch_url
+## patch_url
 
 A url to a patch file for your ruby install.
 
 The default is `nil`.
 
-### patch_file
+## patch_file
 
 A path to a patch file for your ruby install.
 
 The default is `nil`.
 
-### rubies
+## rubies
 
 A list of additional system-wide rubies to be built and installed using the [ruby_build cookbook][ruby_build_cb]. You **must** include `recipe[ruby_build]` in your run_list for the `rbenv_ruby` LWRP to work properly. For example:
 
 ```ruby
 node.default['rbenv']['rubies'] = [ "1.9.3-p0", "jruby-1.6.5" ]
-```
+`
+````
```

Comment:
The README in general will need a total fresh start. I imagine the resources will be completely different.